lib/toc-builder.js

```diff
@@ -131,6 +131,18 @@ function loadFcpData(contentFolder, dataFolder) {
       result.push(fileContents);
     }
   }
+  // the source code for "virtual-module"
+  return `${JSON.stringify(result, null, 2)}`;
+}
+
+function loadIndexData(contentFolder, dataFolder) {
+  const data = loadRFCData(contentFolder, dataFolder);
+
+  const result = [];
+
+  for (let fileContents of data) {
+      result.push({number: fileContents.number, link: fileContents.rfcFile, title: fileContents.title,stage: fileContents.currentStage});
```

Comment:
since we're showing proposed things we won't always have a link to the rfcFile 🤔 should we also be exposing github links? what is the UX around this?

Reply:
Good question, I think we might want to add the GitHub links as a seperate thing (with the open external link icon) so that it's clearer what a link means.